src/registrar/models/portfolio.py

```diff
@@ -0,0 +1,109 @@
+from django.db import models
+
+from registrar.models.domain_request import DomainRequest
+from registrar.models.federal_agency import FederalAgency
+
+from .utility.time_stamped_model import TimeStampedModel
+
+
+def get_default_federal_agency():
```

Comment:
**blocking:** Let's not do this, this is a non class specific function inside of a django model which is not standard practice. Django models should have the info needed for that model and only it. Add this instead as a function to FederalAgency.